controllers/service.go

```diff
@@ -34,9 +34,10 @@ type EmailForm struct {
 }
 
 type SmsForm struct {
-	Content   string   `json:"content"`
-	Receivers []string `json:"receivers"`
-	OrgId     string   `json:"organizationId"` // e.g. "admin/built-in"
+	Content   string          `json:"content"`
+	Receivers []string        `json:"receivers"`
+	OrgId     string          `json:"organizationId"` // e.g. "admin/built-in"
+	Provider  object.Provider `json:"provider"`
```

Comment:
Is provider needed here? If needed, use string in similar way as Email Form:

![image](https://user-images.githubusercontent.com/3787410/222457855-246a8a88-a5b7-4e0a-8405-8cdd2943d349.png)